frontend/app/src/components/asset-manager/ManagedAssetForm.vue

```diff
@@ -79,6 +80,16 @@ const {
   addAsset
 } = useAssetManagementApi();
 
+const v$ = useVuelidate(
+  {
+    asset_type: { required }
+  },
+  {
+    asset_type: computed(() => get(assetType))
+  },
+  { $autoDirty: true }
+);
+
 watch(address, async () => {
   const sanitizedAddress = sanitizeAddress(get(address));
   if (get(address) !== sanitizedAddress) {
```

Comment:
after the `decimals`, `name`, and `symbol` filled, after we search by the address, 
the error messages for these fields need to be cleared.

Reply:
okay, so those 3 are also updated on auto fetching the address...

just noticed that even though `decimals` is set after fetching address, it still says the decimals field cannot be null